src/cli_config.rs

```diff
@@ -113,84 +126,180 @@ pub fn cli_parse() -> CliConfig {
         .author(crate_authors!("\n"))
         .about(crate_description!())
         .setting(AppSettings::ColoredHelp)
-        .arg(Arg::with_name("udp-address")
-            .short("u")
-            .long("udp-address")
-            .help("UDP address to run DHT node")
-            .takes_value(true)
-            .required_unless("tcp-address"))
-        .arg(Arg::with_name("tcp-address")
-            .short("t")
-            .long("tcp-address")
-            .help("TCP address to run TCP relay")
-            .multiple(true)
-            .takes_value(true)
-            .use_delimiter(true)
-            .required_unless("udp-address"))
-        .arg(Arg::with_name("secret-key")
-            .short("s")
-            .long("secret-key")
-            .help("DHT secret key. Note that you should not pass the key via \
+        .subcommand(SubCommand::with_name("config")
+            .arg(Arg::with_name("cfg-file")
```

Comment:
Can you make it positional? To use it like `tox-node config PATH/to.yml ?

Reply:
done